infra/azure/outputs.tf

```diff
@@ -0,0 +1,27 @@
+output "kube_config" {
+  value     = azurerm_kubernetes_cluster.vdc.kube_config_raw
+  sensitive = true
+}
+
+output "global_config" {
+  value = {
+    cloud = "azure",
+    domain = var.domain,
+    docker_prefix = azurerm_container_registry.acr.login_server,
+    internal_ip = local.internal_ip,
```

Comment:
Do we also need resource group and subscription id here? I need that for the AzureComputeClient etc.